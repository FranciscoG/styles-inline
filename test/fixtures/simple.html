<!DOCTYPE html><html><head><title>Simple</title><link rel="stylesheet" href="style.css"></head><body><header></header><main></main></body></html>