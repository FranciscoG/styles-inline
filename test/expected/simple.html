<!DOCTYPE html><html><head><title>Simple</title><style data-href="style.css">
body {
  color:#fff;
  font-family: sans-serif;
}

main {
  width: 500px;
}

a {
  text-decoration: none;
}
</style></head><body><header></header><main></main></body></html>